.gitignore

```diff
@@ -183,10 +183,6 @@ tags
 
 ### VisualStudioCode ###
 .vscode/*
```

Comment:
why `.vscode/*` ?
why not `.vscode/`?
Will be good if you will know the difference between these options.

Reply:
`*` will probably match files but not the folder itself...

Reply:
And I bet it's coming from gitignore.io

Reply:
I think it's used in conjunction with the following file inclusions. Other than that — it doesn't really matter.